### ISSUE TITLE
pretrained/simclr/r101_1x_sk0.pth

### ISSUE THREAD

**Issue by leo-hao** [microsoft/ProDA#38]
Thank you very much for the open source of your code and I am very interested in your work. When inference Using Pretrained Model， I had a problem and I don t know how to choose the simclr weights. [Simclrv2](https://github.com/google-research/simclr)
![image](https://user-images.githubusercontent.com/20486586/141715300-c02b342a-3c5e-4364-80fa-6f8ff41e91da.png)
I enter the Fine-tuned SimCLRv2 models on 100% of labels, but I don't konw which model I should use.
![image](https://user-images.githubusercontent.com/20486586/141715373-63fa52f8-c51e-4f76-9d30-545e96ee1aa9.png)
